package.json

```diff
@@ -14,6 +14,7 @@
     "cleanup-loader": "0.1.1",
     "cross-env": "^5.1.3",
     "css-loader": "^0.28.7",
+    "deep-assign": "^2.0.0",
```

Comment:
Hast du dir das Repo angesehen warum es deprecated ist? Hab nur gesehen:"https://github.com/sindresorhus/deep-assign" --> Ich werd mir das in Ruhe anschauen oder wir suchen eine alternative :) da sollt es genug geben.

Reply:
Sollen wir https://lodash.com/docs/4.17.5#merge verwenden?

Reply:
Lodash ist super